rts/Rendering/GL/VAO.h

```diff
@@ -0,0 +1,48 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#ifndef GL_VAO_H
+#define GL_VAO_H
+
+#include <cstdint>
+#include <utility>
+#include "Rendering/GL/myGL.h"
+
+struct VAO {
+public:
+	static bool IsSupported() {
+		static bool supported = GLEW_ARB_vertex_array_object;
```

Comment:
Why `static bool`? The cost of looking up an exported GLEW symbol is marginal.

Reply:
Shaving off the cost where I can. What's done is done.